config/defaults.mk

```diff
@@ -570,6 +571,17 @@ PARELAG_DIR = @MFEM_DIR@/../parelag
 PARELAG_OPT = -I$(PARELAG_DIR)/src -I$(PARELAG_DIR)/build/src
 PARELAG_LIB = -L$(PARELAG_DIR)/build/src -lParELAG
 
+# Tribol library configuration
+ifeq ($(MFEM_USE_TRIBOL),YES)
+   BASE_FLAGS = -std=c++14
+endif
+AXOM_DIR = @MFEM_DIR@/../axom
+TRIBOL_DIR = @MFEM_DIR@/../tribol
+TRIBOL_OPT = -I$(TRIBOL_DIR)/include -I$(AXOM_DIR)/include -I$(CONDUIT_DIR)/include/conduit
+TRIBOL_LIB = -L$(TRIBOL_DIR)/lib -ltribol -lredecomp -L$(AXOM_DIR)/lib -laxom_mint\
+   -laxom_sidre -laxom_slam -laxom_slic -laxom_core -L$(CONDUIT_DIR)/lib -lconduit\
+   -lconduit_blueprint -lconduit_blueprint_mpi -lconduit_relay -lconduit_relay_mpi
```

Comment:
The `conduit` libs are optional, right?

Reply:
Yes, conduit isn't needed.  It was installed by default by the spack/uberenv build system, but the updated instructions no longer use it (see latest commits).